source/css/_custom.scss

```diff
@@ -466,17 +466,13 @@ ul.participants.list {
     width: 100%;
     overflow: hidden;
     height: 80px;
-    font-size: 24px;
-    line-height: 50px;
-    padding: 10px;
+    font-size: 18px;
     margin: 0;
-    border: solid #eee;
-    border-width: 1px 0;
     text-align: left;
 
     .logo {
       display: table-cell;
```

Comment:
Without the display property being set to `block`, rather than to behave like table cell here, all `.svg` logos without explicit width definition (such as the Applatix's logo) are missing (size 0x0) in the layout for smaller viewports, where the `.participant-name` is hidden.